bundles/af-core/src/test/resources/schema/0.14.2/adaptive-form.schema.json

```diff
@@ -1439,4 +1439,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Please remove these changes

Reply:
Removed, I think this was the culprit, these changes were added by Intellij automatically.